hack/update-generated-crd.sh

```diff
@@ -27,6 +47,10 @@ install_crd \
   "vendor/github.com/openshift/api/operator/v1/zz_generated.crd-manifests/0000_50_ingress_00_ingresscontrollers-TechPreviewNoUpgrade.crd.yaml" \
   "manifests/00-custom-resource-definition-TechPreviewNoUpgrade.yaml"
 
+install_crd \
+  "vendor/github.com/openshift/api/operator/v1/zz_generated.crd-manifests/0000_50_ingress_00_ingresscontrollers.crd.yaml" \
+  "manifests/00-custom-resource-definition.yaml"
```

Comment:
You made this change for the verify script, but you should also make it for the update script:
```suggestion
shopt -s extglob
install_crd \
  vendor/github.com/openshift/api/operator/v1/zz_generated.crd-manifests/0000_50_ingress_00_ingresscontrollers?(-Default).crd.yaml \
  "manifests/00-custom-resource-definition.yaml"
```

Reply:
@Miciah sorry yes done and pushed the changes.